pkg/kube/util/reference/reconciles.go

```diff
@@ -63,14 +63,18 @@ func GetReconciles(ctx context.Context, client crc.Client, reconcileType Reconci
 			return false, errors.Wrap(err, "error listing references")
 		}
 
-		if versionedSecret && reconcileType != ReconcileForExtendedStatefulSet {
+		if versionedSecret {
 			keys := make([]string, len(objectReferences))
 			i := 0
 			for k := range objectReferences {
```

Comment:
I'm always confused by this, shouldn't `for i, num := range objectReferences {` work and take care of the loop variable?